queue/src/fiskaltrust.Middleware.Queue/SignProcessor.cs

```diff
@@ -205,7 +205,12 @@ private async Task<ReceiptResponse> InternalSign(ftQueue queue, ReceiptRequest d
                 {
                     if (queueItem.country != "IT")
                     {
-                        throw exception;
+
+                        var errorMessage = "An error occurred during receipt processing with ftState = 0xEEEE_EEEE for a country other than Italy.";
```

Comment:
```suggestion
                        var errorMessage = "An error occurred during receipt processing, resulting in ftState = 0xEEEE_EEEE.";
```

No need to mention Italy here, this will just confuse users IMO :)